our-process/policies.md

```diff
@@ -65,41 +68,50 @@ external guest editor will be recruited to lead the package review.
 
 ## After Acceptance: Package Ownership and Maintenance
 
-Authors of contributed packages essentially maintain the same ownership they
-had prior to their package joining the pyOpenSci suite. Package authors will
-continue to maintain
-and develop their software after acceptance into pyOpenSci. Unless explicitly added as
-collaborators, the pyOpenSci team will not interfere with day to day operations.
+Package authors are expected to maintain and develop their software and retain
+ownership of it after acceptance into pyOpenSci, as per the peer review agreement
+acknowledged upon submission. This maintenance commitment should last for at
+least two years. The pyOpenSci team will not interfere with day-to-day tool
+maintenance unless explicitly added as collaborators.
 
 If you need to step down from maintaining your accepted pyOpenSci package, please
-please notify the EiC / software review lead of this as soon as you can. We
-will talk with you about your tool and work together to either find a new maintainer or sunset the tool depending upon what makes the most sense.
+promptly notify the pyOpenSci Editor-in-Chief or Software Review Lead. pyOpenSci
+will collaborate with you to either:
+
+- Find a new maintainer or
+- Archive the tool, depending on what best suits your specific scientific Python
+  package.
 
-We will reach out to our package maintainers each year to check in on how
-maintenance is going.
+We will reach out to our package maintainers each year to assess the status of
+maintenance and to see if there are any updates we can highlight through our social channels.
 
-### Maintainer Responsiveness
-If package maintainers do not respond in a timely manner to requests for
-package fixes, we will remind the maintainer a number of times. After 3 months
-(or shorter time frame, depending on how critical the fix is) we will discuss
-the future of the package as a part of our pyOpenSci ecosystem.
+### Maintenance Tracking
 
-* If a package becomes completely un-maintained we will highlight that fact and
-remove it as a vetted tool in our ecosystem.
-* If a sub-community decides to fork and maintain that package we are open to working with the new maintainers to keep the package within our ecosystem.
+pyOpenSci is building a system to track package metrics, including issues, pull requests, and dates of
+the last release and commits to the package repository. Based on activity in
+the repository:
 
-<!-- Should authors abandon the maintenance of an actively used package in our suite,
-we will consider petitioning PyPI to transfer package maintainer status to pyOpenSci. -->
+- Packages not updated within 6 months will receive a flag on the website in the form of a small icon.
```

Comment:
What is an "update"? Any change the repro? A tagged release?

If "update" just means "any activity in the repro e.g. discussion on some issue", I think it's fine. If "update" means "release a tagged version" 6 months is too short. Python itself releases major versions only every 18 months and numpy changes are typically small; 6 month is just one numpy release. So, a stable package without significant bugs might not need an "update" that often. I suggest 12 months.

Reply:
got it. i'm thinking of a way to "watch" for repo activity. do you think if "update" meant either an issue comment or a commit / release 6 months would be reasonable. or do you still think we should wait a full year?

i was thinking something along the lines of 

1. 6 months - no activity - we just have a yellow flag or some shade of green that is not the "good to go" green. 
2. maybe we could then have a 12 months flag that turns red and we check in or maybe it's just orange at that point. 
there's are just markers for us to see what's going on with the package, not to actually archive the package from our ecosystem. thoughts?

Reply:
I think this suggestion is good. "shade of green" after 6 months of "nothing" (no issue, not code change, no release) and then "orange" after 12 month if there is no release (but even if there are issues and PRs. We follow-up manually and decide to give it another year or to archive). I think the decision to "archive" always warrants at least a cursory look by a human.

Reply:
absolutely. i just thought as our community of packages grows (rOpenSci is up to 300) it will be hard to manually track that. so we can automate the initial flags and always manually override as it makes sense. i'll adjust the language just a bit locally and will push a change

Reply:
new language - i'm pushing this up now!

> 
> pyOpenSci is building a system to track package metrics and activity, including issues, pull requests, and dates of
> the last release and last commit to the package repository. Activity is defined as a repository commit, pull request or release. 
> 
> We will 
> flag packages that haven't been updated on a 6-12 month time period based on activity as follows: 
> 
> - Packages with no activity for at least 6 months will receive a green flag on the website indicating that the package is still potentially maintained but no activity has been recorded in that time frame.  
> - Packages with no activity after 12 months will be flagged. At that time, pyOpenSci editorial team member will contact the package maintainers to evaluate the maintenance status of their package.